packages/render/src/use-frame.tsx

```diff
@@ -529,10 +529,14 @@ export function useFrame<
       return;
     }
 
+    const requiredFrameContext = { ...frameContext };
+    // Address is not included in post action
+    delete requiredFrameContext.address;
```

Comment:
would prefer not assigning this property over deleting it tbh - delete is quite rare and mutates, so makes it harder to reason about; generally prefer non mutating JS